traffic_ops/app/db/migrations/2023012316280200_create_new_role.down.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+DELETE FROM public.role_capability rc WHERE  rc.role_id='trouter'
+    ON CONFLICT DO NOTHING;
```

Comment:
Migrations should not have `ON CONFLICT DO NOTHING`, we keep track of migration version the DB is at, and if it runs unnecessarily, we *do* want to see the conflict, because that should never happen.

Reply:
Removed